### DISCUSSION TITLE
代码检查和代码格式化要不要做成硬性的？

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#700] [Ideas]
https://github.com/typicode/husky

例如检查和格式化没过，代码就提不了，或者维持现在的状况，优先保证代码提交，有问题再说

**Comment by mark9804** [comment]
我都可以，看老大哥意思

**Reply by Arondight** [reply]
算了，今天有点时间划水突然飘了，这么搞估计 fork 的人会觉得纯粹是在没事找事，还是把时间用在学习上吧

**Reply by Arondight** [reply]
我在公司内部用了 husky 发现比较傻，打了 pre-commit hook 但是不会提交格式化的结果

**Reply by Arondight** [reply]
hooks 里面写了个 git add 命令，凑合用吧

**Reply by mark9804** [reply]
我这边的可以啊，你参考一下这个 https://github.com/ba-archive/ba-story-player/commit/56038f92bf1339477f68e9fa8eafa856400aa854
